library/manageiq.py

```diff
@@ -97,9 +97,9 @@ def update_provider(self, provider_id, provider_name, endpoints):
         """ Updates the existing provider with new parameters
         """
         try:
-            result = self.client.post(self.providers_url + '/%d' % provider_id,
-                                      action='edit',
-                                      connection_configurations=endpoints)
+            self.client.post(self.providers_url + '/%d' % provider_id,
```

Comment:
Tox is a tool which serves multiple purposes. It can create virtualenvs, install deps and then run the tests. I don't know much about Travis, but Tox can be used to run the tests locally - on your computer. Travis may just run tox.